template/buefy/pages/buefy-sample/buefy-sample.vue

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator';
+import { RequireVue } from '@/modules/core/constants';
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+
+// tslint:disable-next-line:no-var-requires
+const { BDropdown, BDropdownItem } = require('buefy/dist/components/dropdown') as RequireVue;
```

Comment:
Why was this done with a require instead of with the import statement?

Reply:
Import will complain about missing type or sub-path-imports - or both.

This approach loads only a single component from the Buefy library (which is why we don't use Vue.use(buefy)). In the end, the chunk for each page only contains the code needed for those components. Otherwise, all of Buefy gets loaded.